exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -287,20 +303,21 @@ private ExecConstants() {
       PARQUET_WRITER_ENABLE_DICTIONARY_ENCODING,
       new OptionDescription("For internal use. Do not change."));
 
-  public static final String PARQUET_WRITER_USE_PRIMITIVE_TYPES_FOR_DECIMALS
-      = "store.parquet.writer.use_primitive_types_for_decimals";
-  public static final OptionValidator PARQUET_WRITER_USE_PRIMITIVE_TYPES_FOR_DECIMALS_VALIDATOR = new BooleanValidator(
-    PARQUET_WRITER_USE_PRIMITIVE_TYPES_FOR_DECIMALS, null);
+  public static final String PARQUET_WRITER_USE_PRIMITIVE_TYPES_FOR_DECIMALS = "store.parquet.writer.use_primitive_types_for_decimals";
+  public static final OptionValidator PARQUET_WRITER_USE_PRIMITIVE_TYPES_FOR_DECIMALS_VALIDATOR = new BooleanValidator(PARQUET_WRITER_USE_PRIMITIVE_TYPES_FOR_DECIMALS,
+      new OptionDescription("Instructs the Parquet writer to convert decimal to primitive types whenever possible."));
 
-  public static final String PARQUET_WRITER_LOGICAL_TYPE_FOR_DECIMALS
-      = "store.parquet.writer.logical_type_for_decimals";
-  public static final OptionValidator PARQUET_WRITER_LOGICAL_TYPE_FOR_DECIMALS_VALIDATOR
-      = new EnumeratedStringValidator(PARQUET_WRITER_LOGICAL_TYPE_FOR_DECIMALS, null, "fixed_len_byte_array", "binary");
+  public static final String PARQUET_WRITER_LOGICAL_TYPE_FOR_DECIMALS = "store.parquet.writer.logical_type_for_decimals";
+  public static final OptionValidator PARQUET_WRITER_LOGICAL_TYPE_FOR_DECIMALS_VALIDATOR = new EnumeratedStringValidator(PARQUET_WRITER_LOGICAL_TYPE_FOR_DECIMALS,
+      new OptionDescription("Parquet writer logical type for decimal; supported types \'fixed_len_byte_array\' and \'binary\'"),
+      "fixed_len_byte_array", "binary");
 
+  // TODO - The below two options don't seem to be used in the Drill code base
```

Comment:
These were marked as part of @sachouche 's work. We can take these off in the next release or in this PR itself.
@sachouche is it safe to remove this?

Reply:
I think so since I couldn't find any consumer of these options. I guess will know for sure only when we remove them from the code and run all the Drill tests.

Reply:
Ok. I'll mark it as part of DRILL-6527 then.

Reply:
Then it would be good to add `OptionDescription`s  for these options and point there that they are deprecated and will be removed.

Reply:
I'll just add the description as 'Deprecated', since they are anyway not in use in the code anymore.

Reply:
@vvysotskyi / @kkhatua options are considered as part of API (there was a thread in the Drill mailing list). We can remove them only in major releases, i.e. in Drill 2.0. So the below information is not accurate. I have updated DRILL-6527 description, please update `Will be removed starting in 1.15.0` and subsequent comments as well.

Reply:
Done.